libvast/vast/format/bgpdumpbinary.hpp

```diff
@@ -0,0 +1,1455 @@
+#ifndef VAST_FORMAT_BGPDUMPBINARY_HPP
+#define VAST_FORMAT_BGPDUMPBINARY_HPP
+
+#include <istream>
+#include <iostream>
+#include "vast/logger.hpp"
+#include "vast/schema.hpp"
+#include "vast/time.hpp"
+#include "vast/date.hpp"
+#include "vast/address.hpp"
+#include "vast/subnet.hpp"
+#include "vast/data.hpp"
+#include "vast/concept/parseable/core.hpp"
+#include "vast/concept/parseable/string.hpp"
+#include "vast/concept/parseable/numeric/byte.hpp"
+#include "vast/concept/parseable/vast/address.hpp"
+#include "vast/concept/parseable/vast/time.hpp"
+#include "vast/concept/parseable/vast/subnet.hpp"
+#include "vast/concept/parseable/vast/data.hpp"
+#include "vast/concept/printable/vast/address.hpp"
+#include "vast/concept/printable/vast/subnet.hpp"
+#include "vast/concept/printable/vast/data.hpp"
+#include "vast/concept/printable/string.hpp"
+#include "vast/concept/printable/to_string.hpp"
+#include "vast/concept/printable/std/chrono.hpp"
+
+#include "vast/format/reader.hpp"
+
+namespace vast {
+namespace format {
+namespace bgpdumpbinary {
+
+struct form {

```

Comment:
What is a "form"? It would be great if you can summarize the structure of the input this parser is for in the comments. How does the EBNF look like? Where is the format defined? As the reviewer without domain expertise, I'm left clueless where to start.